prow/cmd/phaino/local.go

```diff
@@ -34,10 +35,31 @@ import (
 	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
 )
 
+const (
+	// The well-known env name for gcloud credentials.
+	gcloudCredEnvName = "GOOGLE_APPLICATION_CREDENTIALS"
+	// The path to mount the gcloud default config files to the container.
+	gcloudDefaultConfigMountPath = "/root/.config/gcloud"
+	// The well-known env name for kubeconfig.
+	kubeconfigEnvKey = "KUBECONFIG"
+	// The path to mount the kubectl default config files to the container.
+	kubectlDefaultConfigMountPath = "/root/.kube"
+	// The default GOPATH in the container.
+	GOPATH = "/go"
+	// The default GOSRCPATH in the container.
```

Comment:
Would be nice to also document this is from prow job, aka where cloneref clone source code

Reply:
I read the code and got some interesting findings:
- The root path used by `clonerefs` is `/home/prow/go` by default - https://github.com/kubernetes/test-infra/blob/master/prow/pod-utils/decorate/podspec.go#L52
- The `GOPATH` is set as `/go` in the `kubekins-e2e` image - https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/Dockerfile#L26

I guess the reason that there are no issues due to the inconsistency is, repos do not have to be under GOPATH as long as they are on go modules. But some projects might still be relying on GOPATH, so I have made it configurable and set its default value to `/go`.

Please let me know if it makes sense to you.